src/release/breaking-changes/2-5-deprecations.md

```diff
@@ -0,0 +1,346 @@
+
+const TextFormField textFormField = TextFormField(autovalidate: true);
+const TextFormField textFormField = TextFormField(autovalidate: false);
+
+const DropdownButtonFormField dropDownButtonFormField = DropdownButtonFormField(autovalidate: true);
+const DropdownButtonFormField dropdownButtonFormField = DropdownButtonFormField(autovalidate: false);
+```
+
+Code after migration:
+<!-- skip -->
+```dart
+const Form form = Form(autovalidateMode: AutovalidateMode.always);
+const Form form = Form(autovalidateMode: AutovalidateMode.disabled);
+final autoMode = form.autovalidateMode;
+
+const FormField formField = FormField(autovalidateMode: AutovalidateMode.always);
+const FormField formField = FormField(autovalidateMode: AutovalidateMode.disabled);
+final autoMode = formField.autovalidateMode;
+
+const TextFormField textFormField = TextFormField(autovalidateMode: AutovalidateMode.always);
+const TextFormField textFormField = TextFormField(autovalidateMode: AutovalidateMode.disabled);
+
+const DropdownButtonFormField dropDownButtonFormField = DropdownButtonFormField(autovalidateMode: AutovalidateMode.always);
+const DropdownButtonFormField dropdownButtonFormField = DropdownButtonFormField(autovalidateMode: AutovalidateMode.disabled);
+```
+
+[In-depth migration guide available]: /release/breaking-changes/form-field-autovalidation-api
+
+**References**
+
+API documentation:
+* [`Form`][]
+* [`FormField`][]
+* [`TextFormField`][]
+* [`DropdownButtonFormField`][]
+* [`AutovalidateMode`][]
+
+Relevant issues:
+* [Issue 56363]({{site.repo.flutter}}/issues/56363)
+* [Issue 18885]({{site.repo.flutter}}/issues/18885)
+* [Issue 15404]({{site.repo.flutter}}/issues/15404)
+* [Issue 36154]({{site.repo.flutter}}/issues/36154)
+* [Issue 48876]({{site.repo.flutter}}/issues/48876)
+
+Relevant PRs:
+* Deprecated in [#59766]({{site.repo.flutter}}/pull/59766)
+* Removed in [#90292]({{site.repo.flutter}}/pull/90292)
+
+[`Form`]: {{site.api}}/flutter/widgets/Form-class.html
+[`FormField`]: {{site.api}}/flutter/widgets/FormField-class.html
+[`TextFormField`]: {{site.api}}/flutter/material/TextFormField-class.html
+[`DropdownButtonFormField`]: {{site.api}}/flutter/material/DropdownButtonFormField-class.html
+[`AutovalidateMode`]: {{site.api}}/flutter/widgets/AutovalidateMode-class.html
+
+---
+
+### `FloatingHeaderSnapConfiguration.vsync`
+
+Supported by Flutter Fix: no
+
+The `TickerProvider` `vsync` property of `FloatingHeaderSnapConfiguration` was
+deprecated in v1.19.
+
+The `vsync` for the animation should instead be specified using
+`SliverPersistentHeaderDelegate.vsync`.
+
+**Migration guide**
+
+Code before migration:
+<!-- skip -->
+```dart
+// TODO - no original migration guide
+```
+
+Code after migration:
+<!-- skip -->
+```dart
+// TODO - no original migration guide
+```
+
+**References**
+
+Design document:
+<!-- TODO - doc is spammed, also not listed on website -->
+* [Control SliverPersistentHeader's showOnScreen Behavior][]
```

Comment:
@LongCatIsLooong can you check this design doc? It looks like it has been pretty heavily spammed.

Reply:
Oh gosh someone even changed the background color. Thanks for letting me know, made it readonly.